inference_segmentation.py

```diff
@@ -361,7 +361,7 @@ def main(params: Union[DictConfig, dict]) -> None:
     num_bands = len(bands_requested)
 
     # Default input directory based on default output directory
-    raw_data_csv = get_key_def('raw_data_csv', params['inference'], expected_type=str, to_path=True,
+    raw_data_csv = get_key_def('raw_data_csv', params['inference'], expected_type=Path, to_path=True,
```

Comment:
can "str "or "Path" types be both valid (expected) data types for the given parameter?

Reply:
Yeah but the parameter `to_path` is set to True so at the end you are suppose to have a Path

Reply:
I follow @ms888ekb's thought: it seems a bit confusing that "raw_data_csv" can be a str, but the expected type is "Path". I understand when I look at the code modifications that even if a str is inputted, it will become a Path, but I feel like it could be confusing. Will still approve this PR since this is minor.